app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -186,6 +186,12 @@ public String getInitialUrl() {
     @Override
     public void onPause() {
         super.onPause();
+
+        final IWebView webView = getWebView();
+        if (webView != null) {
+            webView.onPause();
+        }
+
```

Comment:
This shouldn't be needed. WebFragment (parent) already takes care of forwarding the lifecycle methods: https://github.com/mozilla-mobile/focus-android/blob/master/app/src/main/java/org/mozilla/focus/fragment/WebFragment.java#L109